_functions.sh

```diff
@@ -1488,10 +1601,38 @@ do_start() {
   )
 }
 
+do_docker_start(){
+  (
+  do_load_deployment_descriptor  
+  do_start_onlyoffice
+  do_start_ldap
+  do_start_cmis
+  do_start_database
+  do_start_es
+  do_start_chat_server
+
+  # We need this variable for the setenv
+  export DEPLOYMENT_CHAT_SERVER_PORT
+
+  # We need this variable for the setenv
+  export DEPLOYMENT_ONLYOFFICE_HTTP_PORT
+  export DEPLOYMENT_CMIS_HTTP_PORT
```

Comment:
It should be possible this duplicated part here and on the do_default_start by keeping it on the do_start method